app/lib/text_formatter.rb

```diff
@@ -75,7 +75,7 @@ def rewrite
       entity[:indices].first
     end
 
-    result = ''.dup
+    result = +''
```

Comment:
Wouldn't plain `''` work?

Reply:
No, because of `# frozen_string_literal: true`.